### PR TITLE
fix: use ethers instead of txservice on relayer

### PR DESCRIPTION
**Pull request by liu-zhipeng**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

**Review by wanglonghong — APPROVED** [review]

File: packages/agents/relayer/src/bindings/relays/index.ts [hunk 1]
```diff
@@ -113,32 +112,37 @@ export const pollCache = async () => {
           gasPrice: gasPrice.toString(),
         });
 
-        const gasLimit = await txservice.getGasEstimate(+domain, transaction);
+        //const gasLimit = await txservice.getGasEstimate(+domain, transaction);
+        const gasLimit = await rpcProvider.estimateGas(transaction);
```
Comment (LayneHaber) [inline packages/agents/relayer/src/bindings/relays/index.ts:116]: won't we run into more rpc errors without going through the txservice?

**Review by LayneHaber — APPROVED** [review]
if requesting the RPC provider is the only way to fix for x1 then we can merge, but its not a good practice to hit the providers directly

File: packages/agents/relayer/src/bindings/relays/index.ts (hunk 1 above)
Comment (liu-zhipeng) [reply packages/agents/relayer/src/bindings/relays/index.ts:116]: I believe that the latest version of ethersjs manages most of them, but we need to check txservice deeply later. for now current changes will make x1-testnet work properly as I tested

Comment (liu-zhipeng) [reply packages/agents/relayer/src/bindings/relays/index.ts:116]: all good to merge? @LayneHaber

**Pull request merged**